src/main/kotlin/de/danielmaile/mpp/MPP.kt

```diff
@@ -169,15 +179,14 @@ class MPP : JavaPlugin() {
         server.pluginManager.registerEvents(ListenerRift(), this)
         server.pluginManager.registerEvents(ListenerHealer(), this)
         server.pluginManager.registerEvents(ListenerHitman(), this)
-        server.pluginManager.registerEvents(ResourcePackManager(), this)
     }
 
     private fun registerRecipes() {
```

Comment:
Why are we not registering the recipes?

Reply:
Recipes are registered on item register if you read the code for ItemRegistry

Reply:
Could've deleted that one method though, will do